Sources/FormattingHelpers.swift

```diff
@@ -2029,6 +2059,18 @@ extension Formatter {
                 return declarationParser.index(of: .identifier("View"), after: someKeywordIndex) != nil
             }()
 
+            let isSwiftUIPropertyWrapper = mode == .visibility && { () -> Bool in
+                for dynamicProperty in SwiftUIPropertyWrapper.allCases {
+                    if declarationParser.index(
+                        of: .keyword(dynamicProperty.rawValue),
+                        before: declarationTypeTokenIndex
+                    ) != nil {
+                        return true
+                    }
```

Comment:
Are we able to use the existing `modifiersForDeclaration(at:contains:)` helper here? I think attributes are included as a "modifier" considered by that helper

```suggestion
                    if formatter.modifiersForDeclaration(at: declarationTypeTokenIndex, contains: dynamicProperty.rawValue) {
                        return true
                    }
```

Reply:
Alternatively we could use the helper that takes a `contains` closure to avoid the need to iterate through the all of the SwiftUI property wrappers:

```swift
let swiftUIPropertyWrappers: Set<String> = // ...
let isSwiftUIPropertyWrapper = formatter.modifiersForDeclaration(at: declarationTypeTokenIndex, contains: { _, modifier in
  swiftUIPropertyWrappers.contains(modifier)
})
```

This should be more efficient, and seems simpler too

Reply:
Unfortunately, the usage of `formatter.modifiersForDeclaration` was not working for me.

Reply:
Disregard my comment, I was trying to use an incorrect formatter.